mkdocs.yml

```diff
@@ -7,15 +7,15 @@ site_url: 'https://edgexfoundry.github.io/edgex-docs/'
 repo_url: 'https://github.com/edgexfoundry/edgex-go'
 repo_name: 'edgex/edgex-go'
 copyright: 'Copyright &copy; 2020 EdgeX Foundry'
-strict: true
+strict: false
```

Comment:
Was this change intentional?
https://www.mkdocs.org/user-guide/configuration/#strict